src/components/Picker/BasePicker.tsx

```diff
@@ -136,6 +138,18 @@ function BasePicker<TPickerValue>(
         },
     }));
 
+    /**
+     * Windows will reuse the text color of the select for each one of the options
+     * so we need to color accordingly so it doesn't blend with the background.
+     */
+    const itemColor = useMemo(() => {
+        if (getOperatingSystem() === CONST.OS.WINDOWS) {
+            return theme.text;
+        }
+
+        return theme.textLight;
```

Comment:
```suggestion
        if (getOperatingSystem() === CONST.OS.ANDROID) {
            return theme.textLight;
        }

        return theme.pickerOptionsTextColor;
```

Reply:
ok so this ends up fixing only android color and keep other platforms as is now

Reply:
can you apply this change?

Reply:
adding!

Reply:
Also, update comments accordingly